README.md

```diff
@@ -109,19 +199,40 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project star it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank microverse to help me create this portfolio.
+I would like to thank microverse to help me create this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Is this project work?**
+
+  - No, it is template project
+
+- **Is development finished?**
+
+  - Not yet.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [OPTIONAL] Excellent job including your License inside your README file 🚀. However, when I click the link it takes me to a 404 not found page, please kindly ensure to fix this by changing line 236 back to `
This project is [MIT](./MIT.md) licensed.` 🤓.